tests/resources/generator/algebraic_eqn_computed_var_on_rhs/code.py

```diff
@@ -1,6 +1,30 @@
+# The contents of this file was generated from version 0.2.0 of libCellML.
+
 from math import *
 
 
+__version__ = "0.2.0"
```

Comment:
Why `__version__` in Python and `VERSION` in C? All the other (Python) "constants" are in capital letters, so why not be consistent with the version of libCellML?

Reply:
Ok, that's the Python way of specifying a version. Hmm... ok...

Reply:
Yup, it's the way Python generally does it.